src/Sulu/Bundle/AdminBundle/Resources/js/components/Grid/grid.scss

```diff
@@ -2,7 +2,7 @@ $itemMarginRight: 20px;
 
 .grid {
     margin-right: -$itemMarginRight;
-    overflow: hidden;
+    overflow: visible;
```

Comment:
There was actually a reason for it to be `hidden`... Why did you change it?

Reply:
if the text editor is the last item in the grid, that dropdown would be cut off. why should it be hidden?
![screenshot](https://user-images.githubusercontent.com/5758674/48076536-aa8e1a00-e1e5-11e8-9125-a4f6d8bf6210.png)

Reply:
Hm, strange, I thought this was necessary to have not too much space at the end of a form... But it seems like this problem was solved in some other way in the mean time...